tests/packagedcode/data/conan/recipes/boost/manifest/conanfile.py

```diff
@@ -0,0 +1,1831 @@
+from conan import ConanFile
+from conan.errors import ConanException, ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os, to_apple_arch, XCRun
+from conan.tools.build import build_jobs, check_min_cppstd, cross_building, valid_min_cppstd, supported_cppstd
```

Comment:
We can probably remove some unused sections from these huge conanfile.py test files safely without any effect on the parsing.